Commons config update broke Initialize

I saw this error trying to initialize current 1.9 branch:
<pre>
java.util.ServiceConfigurationError: org.apache.accumulo.start.spi.KeywordExecutable: Provider org.apache.accumulo.server.init.Initialize could not be instantiated
	at java.util.ServiceLoader.fail(ServiceLoader.java:232)
	at java.util.ServiceLoader.access$100(ServiceLoader.java:185)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at org.apache.accumulo.start.Main.checkDuplicates(Main.java:237)
	at org.apache.accumulo.start.Main.getExecutables(Main.java:228)
	at org.apache.accumulo.start.Main.main(Main.java:84)
Caused by: java.lang.NoSuchMethodError: org.apache.commons.configuration.MapConfiguration.<init>(Ljava/util/Properties;)V
	at org.apache.accumulo.core.conf.Property.precomputeDefaultValue(Property.java:969)
	at org.apache.accumulo.core.conf.Property.<clinit>(Property.java:1345)
	at org.apache.accumulo.server.init.Initialize.<clinit>(Initialize.java:165)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)

</pre>

--- a/assemble/pom.xml
+++ b/assemble/pom.xml
@@ -48,6 +48,10 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+    </dependency>
+    <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
       <optional>true</optional>
@@ -331,7 +335,7 @@
         </property>
       </activation>
       <properties>
-        <artifactList>commons-math3,commons-vfs2,gson,guava,htrace-core,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
+        <artifactList>commons-configuration,commons-math3,commons-vfs2,gson,guava,htrace-core,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
         <assemblyDescriptor>src/main/assemblies/binary-release.xml</assemblyDescriptor>
         <hadoop.profile>2</hadoop.profile>
       </properties>
@@ -345,7 +349,7 @@
         </property>
       </activation>
       <properties>
-        <artifactList>commons-math3,commons-vfs2,gson,guava,htrace-core,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
+        <artifactList>commons-configuration,commons-math3,commons-vfs2,gson,guava,htrace-core,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
         <assemblyDescriptor>src/main/assemblies/binary-release.xml</assemblyDescriptor>
       </properties>
     </profile>
@@ -375,10 +379,6 @@
         <dependency>
           <groupId>commons-collections</groupId>
           <artifactId>commons-collections</artifactId>
-        </dependency>
-        <dependency>
-          <groupId>commons-configuration</groupId>
-          <artifactId>commons-configuration</artifactId>
         </dependency>
         <dependency>
           <groupId>commons-io</groupId>

--- a/assemble/src/main/assemblies/component.xml
+++ b/assemble/src/main/assemblies/component.xml
@@ -36,6 +36,7 @@
         <include>com.google.protobuf:protobuf-java</include>
         <include>javax.servlet:javax.servlet-api</include>
         <include>jline:jline</include>
+        <include>commons-configuration:commons-configuration</include>
         <include>org.apache.commons:commons-math3</include>
         <include>org.apache.commons:commons-vfs2</include>
         <include>org.apache.thrift:libthrift</include>

